extension/script.css

```diff
@@ -0,0 +1,18 @@
+.ta-button {
+    display: flex;
```

Comment:
Prevent DeArrow's hidden titles from being overriden by moving it to a css style from a class.